### DISCUSSION TITLE
NEMS inputs

### DISCUSSION THREAD

**Discussion by NicholasHRoy** [catalyst-cooperative/pudl#3662] [Ideas]
Wondering if there is any plans to grab from NEMS now that EIA has published the code. There are some costs such as O&M costs or transmission lines which you might be able to find there that previously weren't as accessible.

**Comment by zaneselvans** [comment]
Hey @NicholasHRoy! We were excited to see that all posted too and have also had to pick through their encrypted zipfiles for economic inputs in the past.  We've also recently started bringing into some thematically similar data from the NREL ATB and EIA AEO data for use by modelers.  Is there a set of values which are uniquely available from NEMS that you think we should prioritize if we dig into it?

**Comment by NicholasHRoy** [comment]
Hi, for the NEMS inputs input/emm/emm_db/PLTF860_RDB.xlsx has variables FOM and VOM that could be mapped into the existing catalyst data. As for transmission, my best guess is that input/emm/emm_db/ECPDAT_RDB.xlsx will have the variables needed, but I am not sure about this as much. Please keep me updated as I intend to build our model on catalyst cooperative data and these are the last two missing pieces for us to do that.

**Reply by zaneselvans** [reply]
Wow that's great to hear @NicholasHRoy! I'll add the NEMS FOM/VOM to our data shopping list.

We're talking internally about organizing our institutional users into a cost-sharing pool and advisory group that can ensure the ongoing maintenance expenses for PUDL are covered longer term and that the data we're providing keeps meeting the community's needs, if that's something RFF would be interested in chatting about at some point.

**Reply by NicholasHRoy** [reply]
Once we become users of the data, we would probably be happy to get involved. Using the dataset still might be a few months out, but once we do I'll pitch it. Also looking around for transmission data, I realize this sheet: input/emm/emm_db/PRETTIN_RDB.xlsx is also important for transmission inputs for NEMS.
